Automattic/wp-calypso
# Plugin Details: Split the page structure into smaller components

#### Changes proposed in this Pull Request
* Turn PluginDetailsHeader into a component and use it on the PluginDetails page.
* Turn PluginDetailsCTA into a component and use it on the PluginDetails page.
* Turn PluginDetailsSidebar into a component and use it on the PluginDetails page.
* Rename SiteList component to SiteListArea. 
* Add padding to info divs.
* Remove unnecessary class.

#### Testing instructions
* Open the Plugin Details page (`/plugins/:plugin-slug`) in production.
* Open a new tab with the PluginDetails version of this PR.
* Both should look the same.
* Verify all the actions(CTA, Site list actions) on the also still working.
